healthcare/api-client/fhir/fhir_resources_test.py

```diff
@@ -53,36 +54,55 @@ def test_fhir_store():
 
     yield fhir_store
 
+    # Clean up
     fhir_stores.delete_fhir_store(
         service_account_json, project_id, cloud_region, dataset_id, fhir_store_id
     )
 
 
-@pytest.mark.skip(reason="TODO(noerog): enable when resource updated")
-def test_CRUD_search_resource(test_dataset, test_fhir_store, capsys):
-    response = fhir_resources.create_resource(
+# Fixture that creates/deletes a Patient resource for various tests.
+@pytest.fixture(scope="module")
+def test_patient():
+    patient_response = fhir_resources.create_patient(
         service_account_json,
         base_url,
         project_id,
         cloud_region,
         dataset_id,
         fhir_store_id,
-        resource_type,
     )
+    patient_resource_id = patient_response.json()["id"]
 
-    # Save the resource_id because you need to pass it into later tests
-    resource_id = response.json()["id"]
+    yield patient_resource_id
 
-    fhir_resources.search_resources_get(
+    # Clean up
+    fhir_resources.delete_resource(
         service_account_json,
         base_url,
         project_id,
         cloud_region,
         dataset_id,
         fhir_store_id,
         resource_type,
+        patient_resource_id,
     )
 
+
+def test_CRUD_patient(test_dataset, test_fhir_store, capsys):
```

Comment:
This should be separated into four tests - one for each CRUD operation

Reply:
Done.